### PR TITLE
feat: elaborate on pull requests

### PR DESCRIPTION
**Pull request by fgheysels**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
 - You have a question on the library, how to use it or where to find something
 - You want to raise a concern
 
-Work will be assigned by the code owners of the Arcus library and it is recommended to check with them first diving into code.
-> Also: a good tip is to assign one or every code owner of the repository to your PullRequest.
+**In these cases, we would like you to create an issue**
+- You encounter a problem or bug while using Arcus.
+- You have a clear idea for a feature or enhancement in Arcus.
+
+After you have created an issue, work will will be assigned by the code owners of the Arcus library.  However, nothing you are also encouraged to fix the bug you've reported or implement the feature that you've proposed yourself as well!  In that case, it is recommended to check with the code owners of the Arcus repository first before diving into code.
```
Comment (stijnmoreels) [inline CONTRIBUTING.md:21]: "nothing stops you" probably?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
+
+
+## Pull Requests
+Changes to the code-base are always done via a pull request.
+If you are facing a bug that is really bugging you, or if you're eagerly waiting for some feature to be implemented, you can taken matters into your own hands and fix the bug or implement the feature yourself!
+
+You can do this by **forking** the repository to your personal GitHub account and start working on the forked repository.  Once you've finished implementing the changes, create a pull request to the upstream Arcus repository to make your changes official!
+Arcus contributors will review your pull request.  After the review process is complete, your pull request will be merged by one of the Arcus code owners into the Arcus repository.
```
Comment (stijnmoreels) [inline CONTRIBUTING.md:29]: Code owners will review the pull requests.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+
+> A good tip is to assign one or every code owner of the repository to your pull request.
 
 ## Forking
 Make sure that you have forked the Arcus repository to which you want to contribute to your personal GitHub workplace. We work solely with pull requests (PRs) for new features or adaptations.
 
 > More information on GitHub forking is available [here](https://guides.github.com/activities/forking/).
 
-When working on an issue, we recommand that you create a feature branch on your own GitHub workplace and merge it via pull request into the `master` branch of the destination Arcus repository. That way, you can keep your own `master` branch up to date with the current changes.
```
Comment (stijnmoreels) [inline CONTRIBUTING.md:29]: Maybe the stuff on forking should be before the pull request stuff.

**Review by stijnmoreels — CHANGES_REQUESTED** [review]
Aha aha, great stuff. Happy that someone took the time to fix our sloppy spelling contribution guide 😉👍.

File: CONTRIBUTING.md (hunk 1 above)
Comment (fgheysels) [reply CONTRIBUTING.md:21]: Yes, I was first going to write 'nothing stops you', but then decided to rephrase.  I'll modify this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (fgheysels) [reply CONTRIBUTING.md:29]: Yeah, I thought: anybody can review.

File: CONTRIBUTING.md (hunk 3 above)
Comment (fgheysels) [reply CONTRIBUTING.md:29]: Difficult :)  In the PR part, we're indeed referring to forking.   How it must be done is then explained later.
If you turn it around, you indeed explain what forking is, but then one might wonder:  why do I need to know this.  And the aha-erlebnis comes later.   I think both options are valid.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,32 +1,41 @@
...
 - You have a question on the library, how to use it or where to find something
 - You want to raise a concern
 
-Work will be assigned by the code owners of the Arcus library and it is recommended to check with them first diving into code.
-> Also: a good tip is to assign one or every code owner of the repository to your PullRequest.
+**In these cases, we would like you to create an issue**
+- You encounter a problem or bug while using Arcus.
+- You have a clear idea for a feature or enhancement in Arcus.
+
+After you have created an issue, work will will be assigned by the code owners of the Arcus library.  However, you are also encouraged to fix the bug you've reported or implement the feature that you've proposed yourself as well!  In that case, it is recommended to check with the code owners of the Arcus repository first before diving into code.
```
Comment (stijnmoreels) [inline CONTRIBUTING.md:21]: This is an improvement, Thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (stijnmoreels) [reply CONTRIBUTING.md:29]: Yes, and maybe we should support that, but I'm not sure if we should put that in our contribution guide. Like this, it seems that any contributor could approve and merge a pull request.

File: CONTRIBUTING.md (hunk 3 above)
Comment (stijnmoreels) [reply CONTRIBUTING.md:29]: Yes, you're right! In this case, I would make forking a sub-title of pull requests so it's more clear how they belong together.

**Comment by stijnmoreels** [comment]
Feel free to check my suggestions, @fgheysels 😉

File: CONTRIBUTING.md (hunk 2 above)
Comment (fgheysels) [reply CONTRIBUTING.md:29]: Actually, only code-owners have the rights to merge a pull-request.   In theory, everyone can review, but it's code-owners that have the final word.   I guess that is expressed in that text ?

File: CONTRIBUTING.md (hunk 3 above)
Comment (fgheysels) [reply CONTRIBUTING.md:29]: I'll do that; good idea for adding it as a sub

File: CONTRIBUTING.md (hunk 2 above)
Comment (stijnmoreels) [reply CONTRIBUTING.md:29]: Aha, ok, I didn't saw that. Yes, that will be ok. Thx.

**Comment by stijnmoreels** [comment]
Problem with branch, we'll create follow-up PR.

**Pull request merged**